library/Vanilla/Quill/Blots/Embeds/ExternalBlot.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license https://opensource.org/licenses/GPL-2.0 GPL-2.0
+ */
+
+namespace Vanilla\Quill\Blots\Embeds;
+
+use Gdn;
+use Gdn_Controller;
+use HeadModule;
+use Vanilla\Embeds\EmbedManager;
+
+class ExternalBlot extends AbstractBlockEmbedBlot {
+
+    /** @var EmbedManager */
+    private $embedManager;
+
+    /**
+     * @inheritdoc
+     */
+    public function __construct(array $currentOperation, array $previousOperation, array $nextOperation) {
+        parent::__construct($currentOperation, $previousOperation, $nextOperation);
+
+        /** @var EmbedManager embedManager */
+        $this->embedManager = Gdn::getContainer()->get(EmbedManager::class);
+    }
+
+    /**
+     * Add client-side scripts to the current controller.
+     *
+     * @param array $scripts
+     */
+    private function addScripts(array $scripts) {
```

Comment:
We should be able to get rid of this entirely. These scripts are all being lazy loaded by the client.